frontend/app/src/components/defi/loan/loans/makerdao/ManageWatchers.vue

```diff
@@ -22,7 +22,7 @@ const watchers = computed(() => {
   const { identifier } = get(vault);
   return get(loanWatchers).filter(watcher => {
     const watcherArgs = watcher.args;
-    return watcherArgs.vault_id.includes(identifier);
+    return watcherArgs.vaultId.includes(identifier);
```

Comment:
This fixes are only in bugfixes but have not been pulled into develop right?

Reply:
Yes it hasn't, but I put it here too, because I move/refactor some file.